assets/css/main.scss

```diff
@@ -118,6 +119,26 @@ $why-carousel-offset: 0;
   background-position: right $why-carousel-offset;
 }
 
+#landing-cards {
+
+  padding-bottom: 2rem;
+  
+  .card {
+    min-height: 32rem;
```

Comment:
This doesn't make them of equal height:

![image](https://user-images.githubusercontent.com/4251/98523310-0e84c680-2276-11eb-9ecf-e35c901cade2.png)

Hopefully I'll have some time soon to actually make them the same height by definition, across the site.